src/tests/intellij/test_plugin_info.py

```diff
@@ -0,0 +1,48 @@
+import os
+import time
+
+from src.utils.ocr import find_all_sentence_occurrences
+
+DATA_DIR = os.path.dirname(os.path.dirname(os.path.realpath(__file__))) + "/data/"
+
+
+def test_analysis_intellij(setup_intellij):
```

Comment:
Pls rename the test file to something like test_miscellaneous_items.py and rename the test to test_plugin_info.